packages/application-config/src/schema.ts

```diff
@@ -17,7 +17,13 @@ export interface JSONSchemaForCustomApplicationConfigurationFiles {
   /**
    * See https://docs.commercetools.com/custom-applications/api-reference/application-config#entrypointuripath
    */
-  entryPointUriPath: string;
+  entryPointUriPath: (
+    | {
+        [k: string]: unknown;
+      }
+    | EnvVariablePlaceholder
+  ) &
+    string;
```

Comment:
This looks wrong. See other comment above.

Reply:
Updated the schema for a better types output with [this commit](https://github.com/commercetools/merchant-center-application-kit/pull/2474/commits/ffbac5c71ce9d58e56da2a892296c04c0355bf8e).

Thanks for the guidance.